documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -0,0 +1,393 @@
+
+  ```
+  ClientConfig clientConfig = ClientConfig.builder()
+                .controllerURI("tls://<controller-hostname-or-ip>:9090")
+                .trustStore(Constants.TRUSTSTORE_PATH)
+                ...
+                .build();
+  ```
+
+  2. Installing the CA's certificate in the Java system key store.
+  3. Creating a custom truststore with the CA's certificate and supplying it to the Pravega client application via the
+system properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword`.
+
+For Auth, client-side configuration depends on the `AuthHandler` implementation used. If your server is configured to
+use the default `PasswordAuthHandler`, you may supply the credentials as shown below.
+
+  ```
+  ClientConfig clientConfig = ClientConfig.builder()
+                .controllerURI("tls://<controller-hostname-or-ip>:9090")
+                .trustStore(Constants.TRUSTSTORE_PATH)
+                .credentials(new DefaultCredentials("changeit", "marketinganaylticsapp"))
+                .build();
+
+  ```
+
+#### Hostname Verification
+
+Hostname verification during TLS communications verifies that the DNS name to which the client connects matches the hostname specified in either of the following fields in the server's certificate:
+
+* Common Name (`CN`) in the certificate's `Subject` field
+* One of the `Subject Alternative Names` field entries
+
+If the server certificates have a hostname assigned, you have used IP addresses as endpoints for the services, and those hostnames are not accessible from
+the client nodes, you might need to add mappings of
+IP addresses and DNS/Host names in the client-side operating system hosts file.
+
+Alternatively, you may disable hostname verification by invoking `validateHostName(false)` of the ClientConfig builder. However, we strongly recommended avoiding doing that, especially for production environments.
+
+### Having the TLS and Auth parameters take effect
+
+If you are enabling TLS and Auth in an existing Pravega cluster, having the security parameters take effect simply requires
+restarting the servicesm after making the configuration changes. Any running client applications will need to be restarted as well, after making client-side changes as described earlier.
+
+For fresh deployments, just starting the cluster after configuring TLS and Auth, should be enough.
+
+## Conclusion
+
+In this document, we saw how to enable security in a Pravega cluster running in distributed mode. Specifically, we discussed how to:
+* Generate a CA (if needed),
+* Generate server certificates and keys for Pravega services,
+* Sign the generated certificates using the generated CA,
+* Enable and configure TLS and auth on the server Side,
+* Setup the `ClientConfig` on the client side for communicating with a Pravega, cluster running with TLS and auth enabled, and
+* Have TLS and auth take effect.
```

Comment:
auth format

Reply:
I'm not sure I understand this. If you are referring to consistency, we are now using "auth" everywhere in the document.